### ISSUE TITLE
Overview title in the Contents Block

### ISSUE THREAD

**Issue by ekes** [localgovdrupal/localgov_guides#9]
I think the overview title was in the list of pages in the code prior to refactoring. It's now available separately and probably wants to be added as a (clickable) title at the top of the list of pages in the contents block.

**Comment by stephen-cox** [comment]
The title stuff will need reviewing as they are currently incorrect on guide pages.

The overview page shows the node title and lede above and then 'Overview' as an H2 further down the page. E.g. https://new.croydon.gov.uk/rubbish-and-recycling/bulky-waste-collection

The page shows the node title and lede of the overview page and then the title of the actual page as an H2 further down. E.g.  https://new.croydon.gov.uk/rubbish-and-recycling/bulky-waste-collection/bulky-waste-we-collect

This was implemented using the node class stuff we stripped out in the `getPageTitle` and `getPageDescription` methods. As this is gone we need to think of another way of nodes or modules providing a way to override the page title and lede.

Maybe a good time to review how page titles are done.

Initial thoughts on a better way to do titles is either some sort of hook, which seems a bit D7, or a service that takes the entity type and bundle and returns a title and lede, but there maybe a better way.
